kedro-datasets/setup.py

```diff
@@ -41,8 +41,7 @@ def _collect_requirements(requires):
     "pandas.HDFDataset": [
         PANDAS,
         "tables~=3.6.0; platform_system == 'Windows'",
-        "tables~=3.6, <3.9; platform_system != 'Windows' and python_version<'3.9'",
-        "tables~=3.6; platform_system != 'Windows' and python_version>='3.9'",
+        "tables~=3.6; platform_system != 'Windows'",
```

Comment:
Are these two actually different in practice?

Reply:
Yep, one is capped at <3.7 and the other at <4. However: #426